generator.yml

```diff
@@ -233,7 +237,8 @@ builds:
   # - {<< : *ppc32,      << : *next,             << : *llvm,            boot: true,  llvm_version: *llvm_latest}
   # - {<< : *ppc64,      << : *next,             << : *ppc64_llvm,      boot: true,  llvm_version: *llvm_latest}
   - {<< : *ppc64le,    << : *next,             << : *llvm,            boot: true,  llvm_version: *llvm_latest}
-  - {<< : *riscv,      << : *next,             << : *riscv_llvm_full, boot: true,  llvm_version: *llvm_latest}
+  # riscv: Boot disabled (https://github.com/ClangBuiltLinux/continuous-integration2/issues/72)
+  - {<< : *riscv,      << : *next,             << : *riscv_llvm_full, boot: false, llvm_version: *llvm_latest}
```

Comment:
maybe fold this commit into the one that enabled it? (Should be straightforward via interactive rebase)

Reply:
The reason I did a separate commit was so that it was easy to revert when we wanted to re-enable RISC-V. Now that qemu-binaries and boot-utils have had their pulls merged, I could just update `check_logs.py` to have `--use-cbl-qemu` like I suggested in #72 and avoid disabling the builds altogether.

Reply:
sure, so one more commit on top then?

Reply:
Yes, basically :P

Please see https://github.com/nathanchance/continuous-integration2/commits/image-targets-with-riscv for what it would look like. If you think that is okay, I will push it under this branch so that the pull updates.

Reply:
LGTM